ISSUE_TEMPLATE

```diff
@@ -0,0 +1,40 @@
+Before filing an issue:
+-----------------------
+- Attempt to reproduce the issue using 
+  [this sample application](https://github.com/forcedotcom/ServiceSDK-Android/)
+- Search existing issues, including issues that are closed.
+- Review the "Known Issues" 
+  [section of our release documentation](https://github.com/forcedotcom/ServiceSDK-Android/releases) 
+  for your issue.
+- If this is an implementation specific question please refer to the
+  [Snap-Ins for Android Developer Guide](https://developer.salesforce.com/docs/atlas.en-us.noversion.service_sdk_android.meta/service_sdk_android/servicesdk_android_dev_guide.htm)
+  or the [Service Cloud Snap-Ins API Reference](http://forcedotcom.github.io/ServiceSDK-Android/)
+- This issue tracker is intended for bugs and Android Snap-Ins SDK
+  specific questions. If you're asking a general Android development question,
+  please do so on Stack Overflow.
+
+When reporting a bug:
+-----------------------
+Fill out the sections below, leaving the headers but replacing the content. If
+you're unable to provide certain information, please explain why in the relevant
+section. We may close issues if they do not include sufficient information.
+
+### Issue description
+Describe the issue in detail, including observed and expected behavior.
+
+### Reproduction steps
+Describe how the issue can be reproduced, ideally using 
+[this sample application](https://github.com/forcedotcom/ServiceSDK-Android/).
+Please indicate if the issue was reproduced while the user was authenticated
+or unauthenticated.
```

Comment:
What about something like: `Add any sample code that can help us reproduce the issue.`

Reply:
Good addition, thanks.